index.html

```diff
@@ -1,7 +1,30 @@
-Anna
-About
-I am Code The Dream student now.
-I learn Java Script.
-Experience
-I have a certificate from the completion of courses in HTML and CSS, there are also small sites made as part of the course
-Connect
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title> </title>
```

Comment:
I would add a title text, this is the text that shows on the tab.  Maybe your name.